data_management/tables.py

```diff
@@ -41,14 +41,14 @@ def external_objects_table_data(request):
     size = int(request.GET.get('limit', 25))
     offset = int(request.GET.get('offset', 25))
     search = request.GET.get('search', None)
-    sort = request.GET.get('sort', '') or 'doi_or_unique_name'
+    sort = request.GET.get('sort', '') or 'other_unique_name'
```

Comment:
I think more things are going to have an `identifier` than an `other_unique_name`, so maybe if this is the default sort order it should be:
```suggestion
    sort = request.GET.get('sort', '') or 'identifier'
```

Reply:
Just bumping this - maybe it's fine but it would be good to know?